docs/index.html

```diff
@@ -0,0 +1,291 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Ecosystem website</title>
+    <script type="module" src="https://1.www.s81c.com/common/carbon/web-components/version/v2.0.1/tile.rtl.min.js"></script>
+    <script type="module" src="https://1.www.s81c.com/common/carbon/web-components/version/v2.0.1/button.min.js"></script>
+    <script type="module" src="https://1.www.s81c.com/common/carbon/web-components/tag/v2/latest/list.min.js"></script>
+    <script type="module" src="https://1.www.s81c.com/common/carbon/web-components/tag/v2/latest/tag.min.js"></script>
+    <link rel="stylesheet" href="https://1.www.s81c.com/common/carbon-for-ibm-dotcom/tag/v1/latest/plex.css" />
+    <link rel="stylesheet" href="https://1.www.s81c.com/common/carbon/web-components/tag/v2/latest/themes.css" />
+    <link rel="stylesheet" href="https://1.www.s81c.com/common/carbon/web-components/tag/v2/latest/grid.css" />
+    <link rel="stylesheet" href="style.css">
+</head>
+<body class="cds-theme-zone-white custom-theme">
+
+      <header>
+        <div class="cds--grid layout-lead-space-fixed__container">
+          <img src = "ecosystem-logo.svg" width="350px" alt="Qiskit Ecosystem"/>
+          <p class="title">Explore projects that use or extend Qiskit</p>
+      
+          <cds-button href="https://github.com/qiskit-community/ecosystem">
+            Join the Ecosystem
+          <span slot="icon"
+                style="margin: 2px 0 -2px 0">
+            <svg focusable="false" preserveAspectRatio="xMidYMid meet"
+                  xmlns="http://www.w3.org/2000/svg" fill="currentColor"
+                  width="16" height="16" viewBox="0 0 16 16" aria-hidden="true"
+                  data-v-bdc8f36d="">
+              <path d="M13,14H3c-0.6,0-1-0.4-1-1V3c0-0.6,0.4-1,1-1h5v1H3v10h10V8h1v5C14,13.6,13.6,14,13,14z"></path>
+              <path d="M10 1L10 2 13.3 2 9 6.3 9.7 7 14 2.7 14 6 15 6 15 1z"></path>
+            </svg>
+          </span>
+          </cds-button>
+        </div>
+      </header>
+
+      <div class="content cds--grid layout-lead-space-fixed__container">
+        <h5 class="section">Providers</h5>
```

Comment:
I'm not sure these section titles should be a h5, they look a little small to me. In general we should be following the most accessible type hierarchy, so I think the "Providers" and "Libraries" section title should be larger than the tile titles.

Reply:
Yeah, Eric pointed out the same thing. I will make it look larger than the tile titles in the next commit. At first, I thought that perhaps it's even better to write it in a <p> and not as a heading because if it looks so large, it can be distracting from the tiles.

Reply:
I don't have a strong opinion on the size, but IMO having the `<h2>` smaller than the `<h3>` is OK as we're showing the hierarchy visually through the layout, and I think the project tiles should take visual precedence.

Reply:
Agreed with Frank that it's essential your header choice reflects the information hierarchy. This is "semantic web".

If you want your h2s to visually appear smaller than h3, then I would recommend using a more precise selector. Rather than always making h2 smaller, be more precise with a selector like `.section-header`. It would be an anti-pattern imo to always have `h2` be smaller than `h3` - that would confuse future developers when they try adding an `h2`.

Reply:
Fixed! Thanks for your feedback @frankharkins and @Eric-Arellano.